src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,42 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrders = async () => {
+  return knex('orders');
+};
+
+const getOrderById = async (id) => {
+  if (!id) {
+    throw new HttpError('Id should be a number', 400);
+  }
+
+  try {
+    const orders = await knex('orders AS o')
+      // .select('orders.id as id', 'status', 'quantity', 'name')
+      .select(
+        'o.id as orderId',
+        'o.status as orderStatus',
+        'o.user_id',
+        'oi.quantity',
+        'p.id as productId',
+        'p.name',
+        'p.picture',
+        'p.stock_amount',
+        'p.price',
```

Comment:
you also need is_Favorite because that has to be passed to ProductDetail component....according to the issue #48 
But on the other hand maybe favourite should not be here because it is already disabled and user cannot move it to/from favorites or delete from Shopping Cart.

Reply:
Yes it is grey, as well quanity, however i kept quantity.